src/models/types.ts

```diff
@@ -16,11 +16,13 @@ export interface CivilizationBonus {
 }
 
 export enum Focus {
+	// NOTE: Should these be plural or singular?
 	Archery = 'archery',
 	ArcheryAnti = 'anti-archery',
 	Cavalry = 'cavalry',
 	CavalryAnti = 'anti-cavalry',
 	Defense = 'defense',
+	Elephant = 'elephant',
```

Comment:
Thinking we should go with `Elephantry` for consistency. https://en.wiktionary.org/wiki/elephantry